alexandret/evc-bundle/1.0/config/packages/initial.yaml

```diff
@@ -0,0 +1,4 @@
+alexandre_evc:
```

Comment:
This file seams redundant. 

Pleas remove

Reply:
My bad, the redundant file is now removed
git is rebased too